.config/nextest.toml

```diff
@@ -35,6 +35,9 @@ clickhouse-cluster = { max-threads = 1 }
 # behaviors that conflict with each other.  They need to be run serially.
 live-tests = { max-threads = 1 }
 
+[profile.default]
+default-set = 'all() - package(omicron-live-tests) - package(end-to-end-tests)'
```

Comment:
Id recommend also adding a live-tests profile with exactly 'package(omicron-live-tests)' as the default set, so folks can run this with -P live-tests.